src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
```

Comment:
```suggestion
  shape = 'triangle';
```
- The type for `shape` is already set as this class implements Figure /
- You can set value here, as you don't get it from arguments in constructor

Fix it in all places